### PR TITLE
MGDOBR-1004 : Maintain single Kubernetes profile

### PR DESCRIPTION
**Pull request by vaibhavjainwiz**
https://issues.redhat.com/browse/MGDOBR-1004

## PR owner checklist

Please make sure that your PR meets the following requirements:

- [ ] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config).  
  *Run `mvn clean verify -DskipTests` so that imports are correctly set.*
- [ ] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [ ] The layers in the `kustomize` folder have been updated accordingly.
- [ ] All new functionality is tested
- [ ] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

## PR reviewer(s) checklist

- [ ] Check if code and Github action workflows have been modified and if the modification is safe
- [ ] If the modification is safe, add the `safe to test` label

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

- <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

- <b>Rebase the pull request</b>  
  Comment with: `/rebase`.
- <b>Deploy BOT</b>
  Comment with `/deploy <dev,stable> when the PR has been merged to deploy to a target environment.

</details>

### CODE REVIEW

**Review by r00ta — COMMENTED** [review]
I guess you need to update `shard-run.sh` to specify the properties you removed from `application.properties` like the webhook account id

**Comment by vaibhavjainwiz** [comment]
> I guess you need to update `shard-run.sh` to specify the properties you removed from `application.properties` like the webhook account id

done

**Comment by vaibhavjainwiz** [comment]
depends on https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/1072

File: dev/bin/shard-run.sh [hunk 1]
```diff
@@ -15,6 +15,9 @@ mvn \
   -Dminikubeip=${MINIKUBE_IP} \
   -Dquarkus.http.port=1337 \
   -Devent-bridge.logging.json=false \
-  -Pminikube \
+  -Devent-bridge.k8s.orchestrator=minikube \
```
Comment (manstis) [inline dev/bin/shard-run.sh:18]: Do we need to set [this](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/1067/files#diff-d34f9ea355d51ca9a2541fc7489e774973ebd6850c2dca32ff12f8c319bb6fb5L29) too?

I see it removed but not set elsewhere.

Comment (vaibhavjainwiz) [reply dev/bin/shard-run.sh:18]: no, its not required. Thanks for point it out.
I am removing it

Comment (manstis) [reply dev/bin/shard-run.sh:18]: I was referring to `quarkus.kubernetes.deployment-target` that you have deleted from the `application.properties` file but not added back anywhere else. The "this" above is a link to where you delete the property.

Comment (vaibhavjainwiz) [reply dev/bin/shard-run.sh:18]: Now we are only have one single profile `kubernetes` so `quarkus.kubernetes.deployment-target` property is not required.

**Comment by vaibhavjainwiz** [comment]
closing in favour of https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/1079

**Pull request closed without merging**